### DISCUSSION TITLE
Can I enter a repository password mannually?

### DISCUSSION THREAD

**Discussion by ghost** [creativeprojects/resticprofile#241] [Q&A] [answered]
I want to manually enter the repository password when I run commands, so I do not write `password-file` in the configuration file. However it does not give me a chance to enter a password.

```
using configuration file: profiles.toml
profile 'default': starting 'snapshots'
reading repository password from stdin
Fatal: an empty password is not a password
snapshots on profile 'default': exit status 1
```

**Comment by creativeprojects** [comment, marked as answer]
resticprofile was designed with automation in mind: the idea is to write a configuration file instead of a shell script.

For now it's not possible to type a password 😞  You can use the `RESTIC_PASSWORD` environment variable if that works for you?

(Related to: https://github.com/creativeprojects/resticprofile/issues/71)
